<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>343</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>494</width>
    <height>343</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>652</width>
    <height>364</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>SeaWars</string>
  </property>
  <property name="windowIcon">
   <iconset resource="source.qrc">
    <normaloff>:/images/miss.png</normaloff>:/images/miss.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">*#centralwidget{background-image: url(:/images/field.png);background-repeat:none;}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>471</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_Msg">
       <property name="text">
        <string>Сообщение противнику:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="msg_edit">
       <property name="maxLength">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="pin"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>0</y>
      <width>201</width>
      <height>171</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="msg_box">
     <property name="spacing">
      <number>1</number>
     </property>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Меню</string>
    </property>
    <addaction name="act_server"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="act_EndPlacing"/>
   <addaction name="act_Shoot"/>
  </widget>
  <action name="act_server">
   <property name="text">
    <string>Подключиться к серверу</string>
   </property>
   <property name="toolTip">
    <string>Запускает новую игру, заново подключаясь к серверу</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="act_EndPlacing">
   <property name="text">
    <string>Завершить расстановку(Ctrl+R)</string>
   </property>
   <property name="toolTip">
    <string>Нажимайте если корабли расставлены и вы готовы к бою!</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="act_Shoot">
   <property name="text">
    <string>Стрелять (F2)</string>
   </property>
   <property name="toolTip">
    <string>Сначала выберите место на карте противника, затем стреляйте!</string>
   </property>
   <property name="shortcut">
    <string>F2</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="source.qrc"/>
 </resources>
 <connections/>
</ui>
